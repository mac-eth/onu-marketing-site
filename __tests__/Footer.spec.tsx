import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import NavbarStructure from "../src/components/footer/footerStructure";

describe("NavbarStructure", () => {
  it("Renders the component unchanged", () => {
    const aboutFeatures = render(<NavbarStructure />);
    expect(aboutFeatures).toMatchSnapshot();
  });

  it("renders the expected elements", () => {
    render(<NavbarStructure />);
    const footer = screen.getByRole("contentinfo", { name: "footer" });
    expect(footer).toBeInTheDocument();
  });

  it("renders the expected social media icons", () => {
    render(<NavbarStructure />);
    const discordIcon = screen.getByRole("button", { name: "discord" });
    expect(discordIcon).toBeInTheDocument();

    const twitterIcon = screen.getByRole("button", { name: "twitter" });
    expect(twitterIcon).toBeInTheDocument();

    const githubIcon = screen.getByRole("button", { name: "github" });
    expect(githubIcon).toBeInTheDocument();
  });

  it("renders the expected copyright notice", () => {
    render(<NavbarStructure />);
    const copyright = screen.getByText("Â© Onu, Inc. All rights reserved.");
    expect(copyright).toBeInTheDocument();
  });
});
