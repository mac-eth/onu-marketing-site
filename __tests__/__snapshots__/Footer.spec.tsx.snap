// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarStructure Renders the component unchanged 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        aria-label="footer"
        class="bg-white"
      >
        <div
          class="mx-auto max-w-7xl py-12 px-4 sm:px-6 md:flex md:items-center md:justify-between lg:px-8"
        >
          <div
            class="flex justify-center space-x-6 md:order-2"
          >
            <a
              aria-label="discord"
              class="social-icon hover:scale-110 hover:cursor-pointer"
              role="button"
              style="display: inline-block; width: 50px; height: 50px; position: relative; overflow: hidden; vertical-align: middle;"
            >
              <div
                class="social-container"
                style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
              >
                <svg
                  class="social-svg"
                  style="border-radius: 50%; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; fill-rule: evenodd;"
                  viewBox="0 0 64 64"
                >
                  <g
                    class="social-svg-background"
                    style="transition: fill 170ms ease-in-out; fill: transparent;"
                  >
                    <circle
                      cx="32"
                      cy="32"
                      r="31"
                    />
                  </g>
                  <g
                    class="social-svg-icon"
                    style="transition: fill 170ms ease-in-out; fill: transparent;"
                  >
                    <path
                      d="M 0,0 H 64 V 64 H 0 Z"
                    />
                  </g>
                  <g
                    class="social-svg-mask"
                    style="transition: fill 170ms ease-in-out; fill: #7289DA;"
                  >
                    <path
                      d="M 0 0 L 0 64 L 64 64 L 64 0 L 0 0 z M 26.404297 16.828125 L 26.769531 17.259766 C 20.186671 19.137498 17.152344 21.990234 17.152344 21.990234 C 17.152344 21.990234 17.957407 21.557233 19.310547 20.943359 C 23.223697 19.246178 26.331328 18.778253 27.611328 18.669922 C 27.830758 18.633812 28.012992 18.597656 28.232422 18.597656 C 30.463282 18.308769 32.987954 18.236508 35.621094 18.525391 C 39.095384 18.922604 42.825476 19.931951 46.628906 21.990234 C 46.628906 21.990234 43.738624 19.282029 37.521484 17.404297 L 38.035156 16.828125 C 38.035156 16.828125 43.044257 16.719236 48.310547 20.619141 C 48.310547 20.619141 53.576172 30.044365 53.576172 41.671875 C 53.576172 41.671875 50.468999 46.907386 42.386719 47.160156 C 42.386719 47.160156 41.069796 45.606565 39.972656 44.234375 C 44.763516 42.898295 46.591797 39.9375 46.591797 39.9375 C 45.092367 40.91248 43.666719 41.598919 42.386719 42.068359 C 40.558139 42.826669 38.802844 43.332214 37.083984 43.621094 C 33.573124 44.271074 30.354188 44.091468 27.611328 43.585938 C 25.526758 43.188727 23.735758 42.610963 22.236328 42.033203 C 21.395188 41.708213 20.480696 41.310228 19.566406 40.804688 C 19.456696 40.732487 19.346048 40.69722 19.236328 40.625 C 19.163228 40.5889 19.126414 40.551755 19.089844 40.515625 C 18.431554 40.154535 18.066406 39.902344 18.066406 39.902344 C 18.066406 39.902344 19.822217 42.789919 24.466797 44.162109 C 23.369647 45.534299 22.015625 47.160156 22.015625 47.160156 C 13.933335 46.907386 10.861328 41.671875 10.861328 41.671875 C 10.861328 30.044375 16.128906 20.619141 16.128906 20.619141 C 21.395196 16.719236 26.404297 16.828125 26.404297 16.828125 z M 25.380859 30.296875 C 23.296289 30.296875 21.650391 32.101957 21.650391 34.304688 C 21.650391 36.507407 23.332859 38.3125 25.380859 38.3125 C 27.465429 38.3125 29.111328 36.507407 29.111328 34.304688 C 29.147928 32.101957 27.465429 30.296875 25.380859 30.296875 z M 38.728516 30.296875 C 36.643946 30.296875 34.998047 32.101957 34.998047 34.304688 C 34.998047 36.507407 36.680516 38.3125 38.728516 38.3125 C 40.813086 38.3125 42.458984 36.507407 42.458984 34.304688 C 42.458984 32.101957 40.813086 30.296875 38.728516 30.296875 z "
                    />
                  </g>
                </svg>
              </div>
            </a>
            <a
              aria-label="twitter"
              class="social-icon hover:scale-110 hover:cursor-pointer"
              role="button"
              style="display: inline-block; width: 50px; height: 50px; position: relative; overflow: hidden; vertical-align: middle;"
            >
              <div
                class="social-container"
                style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
              >
                <svg
                  class="social-svg"
                  style="border-radius: 50%; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; fill-rule: evenodd;"
                  viewBox="0 0 64 64"
                >
                  <g
                    class="social-svg-background"
                    style="transition: fill 170ms ease-in-out; fill: transparent;"
                  >
                    <circle
                      cx="32"
                      cy="32"
                      r="31"
                    />
                  </g>
                  <g
                    class="social-svg-icon"
                    style="transition: fill 170ms ease-in-out; fill: transparent;"
                  >
                    <path
                      d="m 48.1,22.1 c -1.223105,0.430685 -2.4,0.761371 -3.8,1 1.4,-0.8 2.4,-2.1 2.9,-3.6 -1.3,0.8 -2.7,1.3 -4.2,1.6 -1.2,-1.3 -3,-2.1 -4.8,-2.1 -3.6,0 -6.6,2.9 -6.6,6.6 0,0.5 0.1,1 0.2,1.5 -5.5,-0.3 -10.3,-2.9 -13.5,-6.9 -0.6,1 -0.9,2.1 -0.9,3.3 0,2.3 1.2,4.3 2.9,5.5 -1.1,0 -2.1,-0.3 -3,-0.8 v 0.1 c 0,3.2 2.3,5.8 5.3,6.4 -0.6,0.1 -1.1,0.2 -1.7,0.2 -0.4,0 -0.8,0 -1.2,-0.1 0.8,2.6 3.3,4.5 6.1,4.6 -2.2,1.8 -5.1,2.8 -8.2,2.8 -0.5,0 -1.1,0 -1.6,-0.1 C 18.9,44 22.4,45 26.1,45 38.2,45 44.43575,34.996634 44.67774,26.3 L 44.7,25.5 c 1.2,-1 2.5,-2.1 3.4,-3.4 z"
                    />
                  </g>
                  <g
                    class="social-svg-mask"
                    style="transition: fill 170ms ease-in-out; fill: #00aced;"
                  >
                    <path
                      d="M0,0v64h64V0H0z M44.7,25.5c0,0.3,0,0.6,0,0.8C44.7,35,38.1,45,26.1,45c-3.7,0-7.2-1.1-10.1-2.9 c0.5,0.1,1,0.1,1.6,0.1c3.1,0,5.9-1,8.2-2.8c-2.9-0.1-5.3-2-6.1-4.6c0.4,0.1,0.8,0.1,1.2,0.1c0.6,0,1.2-0.1,1.7-0.2 c-3-0.6-5.3-3.3-5.3-6.4c0,0,0-0.1,0-0.1c0.9,0.5,1.9,0.8,3,0.8c-1.8-1.2-2.9-3.2-2.9-5.5c0-1.2,0.3-2.3,0.9-3.3 c3.2,4,8.1,6.6,13.5,6.9c-0.1-0.5-0.2-1-0.2-1.5c0-3.6,2.9-6.6,6.6-6.6c1.9,0,3.6,0.8,4.8,2.1c1.5-0.3,2.9-0.8,4.2-1.6 c-0.5,1.5-1.5,2.8-2.9,3.6c1.3-0.2,2.6-0.5,3.8-1C47.1,23.4,46,24.5,44.7,25.5z"
                    />
                  </g>
                </svg>
              </div>
            </a>
            <a
              aria-label="github"
              class="social-icon hover:scale-110 hover:cursor-pointer"
              role="button"
              style="display: inline-block; width: 50px; height: 50px; position: relative; overflow: hidden; vertical-align: middle;"
            >
              <div
                class="social-container"
                style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
              >
                <svg
                  class="social-svg"
                  style="border-radius: 50%; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; fill-rule: evenodd;"
                  viewBox="0 0 64 64"
                >
                  <g
                    class="social-svg-background"
                    style="transition: fill 170ms ease-in-out; fill: transparent;"
                  >
                    <circle
                      cx="32"
                      cy="32"
                      r="31"
                    />
                  </g>
                  <g
                    class="social-svg-icon"
                    style="transition: fill 170ms ease-in-out; fill: transparent;"
                  >
                    <path
                      d="M32,16c-8.8,0-16,7.2-16,16c0,7.1,4.6,13.1,10.9,15.2 c0.8,0.1,1.1-0.3,1.1-0.8c0-0.4,0-1.4,0-2.7c-4.5,1-5.4-2.1-5.4-2.1c-0.7-1.8-1.8-2.3-1.8-2.3c-1.5-1,0.1-1,0.1-1 c1.6,0.1,2.5,1.6,2.5,1.6c1.4,2.4,3.7,1.7,4.7,1.3c0.1-1,0.6-1.7,1-2.1c-3.6-0.4-7.3-1.8-7.3-7.9c0-1.7,0.6-3.2,1.6-4.3 c-0.2-0.4-0.7-2,0.2-4.2c0,0,1.3-0.4,4.4,1.6c1.3-0.4,2.6-0.5,4-0.5c1.4,0,2.7,0.2,4,0.5c3.1-2.1,4.4-1.6,4.4-1.6 c0.9,2.2,0.3,3.8,0.2,4.2c1,1.1,1.6,2.5,1.6,4.3c0,6.1-3.7,7.5-7.3,7.9c0.6,0.5,1.1,1.5,1.1,3c0,2.1,0,3.9,0,4.4 c0,0.4,0.3,0.9,1.1,0.8C43.4,45.1,48,39.1,48,32C48,23.2,40.8,16,32,16z"
                    />
                  </g>
                  <g
                    class="social-svg-mask"
                    style="transition: fill 170ms ease-in-out; fill: #24292e;"
                  >
                    <path
                      d="M0,0v64h64V0H0z M37.1,47.2c-0.8,0.2-1.1-0.3-1.1-0.8c0-0.5,0-2.3,0-4.4c0-1.5-0.5-2.5-1.1-3 c3.6-0.4,7.3-1.7,7.3-7.9c0-1.7-0.6-3.2-1.6-4.3c0.2-0.4,0.7-2-0.2-4.2c0,0-1.3-0.4-4.4,1.6c-1.3-0.4-2.6-0.5-4-0.5 c-1.4,0-2.7,0.2-4,0.5c-3.1-2.1-4.4-1.6-4.4-1.6c-0.9,2.2-0.3,3.8-0.2,4.2c-1,1.1-1.6,2.5-1.6,4.3c0,6.1,3.7,7.5,7.3,7.9 c-0.5,0.4-0.9,1.1-1,2.1c-0.9,0.4-3.2,1.1-4.7-1.3c0,0-0.8-1.5-2.5-1.6c0,0-1.6,0-0.1,1c0,0,1,0.5,1.8,2.3c0,0,0.9,3.1,5.4,2.1 c0,1.3,0,2.3,0,2.7c0,0.4-0.3,0.9-1.1,0.8C20.6,45.1,16,39.1,16,32c0-8.8,7.2-16,16-16c8.8,0,16,7.2,16,16 C48,39.1,43.4,45.1,37.1,47.2z"
                    />
                  </g>
                </svg>
              </div>
            </a>
          </div>
          <div
            class="mt-8 md:order-1 md:mt-0"
          >
            <p
              class="text-center text-base text-gray-400"
            >
              Â© Onu, Inc. All rights reserved.
            </p>
          </div>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      aria-label="footer"
      class="bg-white"
    >
      <div
        class="mx-auto max-w-7xl py-12 px-4 sm:px-6 md:flex md:items-center md:justify-between lg:px-8"
      >
        <div
          class="flex justify-center space-x-6 md:order-2"
        >
          <a
            aria-label="discord"
            class="social-icon hover:scale-110 hover:cursor-pointer"
            role="button"
            style="display: inline-block; width: 50px; height: 50px; position: relative; overflow: hidden; vertical-align: middle;"
          >
            <div
              class="social-container"
              style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
            >
              <svg
                class="social-svg"
                style="border-radius: 50%; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; fill-rule: evenodd;"
                viewBox="0 0 64 64"
              >
                <g
                  class="social-svg-background"
                  style="transition: fill 170ms ease-in-out; fill: transparent;"
                >
                  <circle
                    cx="32"
                    cy="32"
                    r="31"
                  />
                </g>
                <g
                  class="social-svg-icon"
                  style="transition: fill 170ms ease-in-out; fill: transparent;"
                >
                  <path
                    d="M 0,0 H 64 V 64 H 0 Z"
                  />
                </g>
                <g
                  class="social-svg-mask"
                  style="transition: fill 170ms ease-in-out; fill: #7289DA;"
                >
                  <path
                    d="M 0 0 L 0 64 L 64 64 L 64 0 L 0 0 z M 26.404297 16.828125 L 26.769531 17.259766 C 20.186671 19.137498 17.152344 21.990234 17.152344 21.990234 C 17.152344 21.990234 17.957407 21.557233 19.310547 20.943359 C 23.223697 19.246178 26.331328 18.778253 27.611328 18.669922 C 27.830758 18.633812 28.012992 18.597656 28.232422 18.597656 C 30.463282 18.308769 32.987954 18.236508 35.621094 18.525391 C 39.095384 18.922604 42.825476 19.931951 46.628906 21.990234 C 46.628906 21.990234 43.738624 19.282029 37.521484 17.404297 L 38.035156 16.828125 C 38.035156 16.828125 43.044257 16.719236 48.310547 20.619141 C 48.310547 20.619141 53.576172 30.044365 53.576172 41.671875 C 53.576172 41.671875 50.468999 46.907386 42.386719 47.160156 C 42.386719 47.160156 41.069796 45.606565 39.972656 44.234375 C 44.763516 42.898295 46.591797 39.9375 46.591797 39.9375 C 45.092367 40.91248 43.666719 41.598919 42.386719 42.068359 C 40.558139 42.826669 38.802844 43.332214 37.083984 43.621094 C 33.573124 44.271074 30.354188 44.091468 27.611328 43.585938 C 25.526758 43.188727 23.735758 42.610963 22.236328 42.033203 C 21.395188 41.708213 20.480696 41.310228 19.566406 40.804688 C 19.456696 40.732487 19.346048 40.69722 19.236328 40.625 C 19.163228 40.5889 19.126414 40.551755 19.089844 40.515625 C 18.431554 40.154535 18.066406 39.902344 18.066406 39.902344 C 18.066406 39.902344 19.822217 42.789919 24.466797 44.162109 C 23.369647 45.534299 22.015625 47.160156 22.015625 47.160156 C 13.933335 46.907386 10.861328 41.671875 10.861328 41.671875 C 10.861328 30.044375 16.128906 20.619141 16.128906 20.619141 C 21.395196 16.719236 26.404297 16.828125 26.404297 16.828125 z M 25.380859 30.296875 C 23.296289 30.296875 21.650391 32.101957 21.650391 34.304688 C 21.650391 36.507407 23.332859 38.3125 25.380859 38.3125 C 27.465429 38.3125 29.111328 36.507407 29.111328 34.304688 C 29.147928 32.101957 27.465429 30.296875 25.380859 30.296875 z M 38.728516 30.296875 C 36.643946 30.296875 34.998047 32.101957 34.998047 34.304688 C 34.998047 36.507407 36.680516 38.3125 38.728516 38.3125 C 40.813086 38.3125 42.458984 36.507407 42.458984 34.304688 C 42.458984 32.101957 40.813086 30.296875 38.728516 30.296875 z "
                  />
                </g>
              </svg>
            </div>
          </a>
          <a
            aria-label="twitter"
            class="social-icon hover:scale-110 hover:cursor-pointer"
            role="button"
            style="display: inline-block; width: 50px; height: 50px; position: relative; overflow: hidden; vertical-align: middle;"
          >
            <div
              class="social-container"
              style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
            >
              <svg
                class="social-svg"
                style="border-radius: 50%; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; fill-rule: evenodd;"
                viewBox="0 0 64 64"
              >
                <g
                  class="social-svg-background"
                  style="transition: fill 170ms ease-in-out; fill: transparent;"
                >
                  <circle
                    cx="32"
                    cy="32"
                    r="31"
                  />
                </g>
                <g
                  class="social-svg-icon"
                  style="transition: fill 170ms ease-in-out; fill: transparent;"
                >
                  <path
                    d="m 48.1,22.1 c -1.223105,0.430685 -2.4,0.761371 -3.8,1 1.4,-0.8 2.4,-2.1 2.9,-3.6 -1.3,0.8 -2.7,1.3 -4.2,1.6 -1.2,-1.3 -3,-2.1 -4.8,-2.1 -3.6,0 -6.6,2.9 -6.6,6.6 0,0.5 0.1,1 0.2,1.5 -5.5,-0.3 -10.3,-2.9 -13.5,-6.9 -0.6,1 -0.9,2.1 -0.9,3.3 0,2.3 1.2,4.3 2.9,5.5 -1.1,0 -2.1,-0.3 -3,-0.8 v 0.1 c 0,3.2 2.3,5.8 5.3,6.4 -0.6,0.1 -1.1,0.2 -1.7,0.2 -0.4,0 -0.8,0 -1.2,-0.1 0.8,2.6 3.3,4.5 6.1,4.6 -2.2,1.8 -5.1,2.8 -8.2,2.8 -0.5,0 -1.1,0 -1.6,-0.1 C 18.9,44 22.4,45 26.1,45 38.2,45 44.43575,34.996634 44.67774,26.3 L 44.7,25.5 c 1.2,-1 2.5,-2.1 3.4,-3.4 z"
                  />
                </g>
                <g
                  class="social-svg-mask"
                  style="transition: fill 170ms ease-in-out; fill: #00aced;"
                >
                  <path
                    d="M0,0v64h64V0H0z M44.7,25.5c0,0.3,0,0.6,0,0.8C44.7,35,38.1,45,26.1,45c-3.7,0-7.2-1.1-10.1-2.9 c0.5,0.1,1,0.1,1.6,0.1c3.1,0,5.9-1,8.2-2.8c-2.9-0.1-5.3-2-6.1-4.6c0.4,0.1,0.8,0.1,1.2,0.1c0.6,0,1.2-0.1,1.7-0.2 c-3-0.6-5.3-3.3-5.3-6.4c0,0,0-0.1,0-0.1c0.9,0.5,1.9,0.8,3,0.8c-1.8-1.2-2.9-3.2-2.9-5.5c0-1.2,0.3-2.3,0.9-3.3 c3.2,4,8.1,6.6,13.5,6.9c-0.1-0.5-0.2-1-0.2-1.5c0-3.6,2.9-6.6,6.6-6.6c1.9,0,3.6,0.8,4.8,2.1c1.5-0.3,2.9-0.8,4.2-1.6 c-0.5,1.5-1.5,2.8-2.9,3.6c1.3-0.2,2.6-0.5,3.8-1C47.1,23.4,46,24.5,44.7,25.5z"
                  />
                </g>
              </svg>
            </div>
          </a>
          <a
            aria-label="github"
            class="social-icon hover:scale-110 hover:cursor-pointer"
            role="button"
            style="display: inline-block; width: 50px; height: 50px; position: relative; overflow: hidden; vertical-align: middle;"
          >
            <div
              class="social-container"
              style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
            >
              <svg
                class="social-svg"
                style="border-radius: 50%; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; fill-rule: evenodd;"
                viewBox="0 0 64 64"
              >
                <g
                  class="social-svg-background"
                  style="transition: fill 170ms ease-in-out; fill: transparent;"
                >
                  <circle
                    cx="32"
                    cy="32"
                    r="31"
                  />
                </g>
                <g
                  class="social-svg-icon"
                  style="transition: fill 170ms ease-in-out; fill: transparent;"
                >
                  <path
                    d="M32,16c-8.8,0-16,7.2-16,16c0,7.1,4.6,13.1,10.9,15.2 c0.8,0.1,1.1-0.3,1.1-0.8c0-0.4,0-1.4,0-2.7c-4.5,1-5.4-2.1-5.4-2.1c-0.7-1.8-1.8-2.3-1.8-2.3c-1.5-1,0.1-1,0.1-1 c1.6,0.1,2.5,1.6,2.5,1.6c1.4,2.4,3.7,1.7,4.7,1.3c0.1-1,0.6-1.7,1-2.1c-3.6-0.4-7.3-1.8-7.3-7.9c0-1.7,0.6-3.2,1.6-4.3 c-0.2-0.4-0.7-2,0.2-4.2c0,0,1.3-0.4,4.4,1.6c1.3-0.4,2.6-0.5,4-0.5c1.4,0,2.7,0.2,4,0.5c3.1-2.1,4.4-1.6,4.4-1.6 c0.9,2.2,0.3,3.8,0.2,4.2c1,1.1,1.6,2.5,1.6,4.3c0,6.1-3.7,7.5-7.3,7.9c0.6,0.5,1.1,1.5,1.1,3c0,2.1,0,3.9,0,4.4 c0,0.4,0.3,0.9,1.1,0.8C43.4,45.1,48,39.1,48,32C48,23.2,40.8,16,32,16z"
                  />
                </g>
                <g
                  class="social-svg-mask"
                  style="transition: fill 170ms ease-in-out; fill: #24292e;"
                >
                  <path
                    d="M0,0v64h64V0H0z M37.1,47.2c-0.8,0.2-1.1-0.3-1.1-0.8c0-0.5,0-2.3,0-4.4c0-1.5-0.5-2.5-1.1-3 c3.6-0.4,7.3-1.7,7.3-7.9c0-1.7-0.6-3.2-1.6-4.3c0.2-0.4,0.7-2-0.2-4.2c0,0-1.3-0.4-4.4,1.6c-1.3-0.4-2.6-0.5-4-0.5 c-1.4,0-2.7,0.2-4,0.5c-3.1-2.1-4.4-1.6-4.4-1.6c-0.9,2.2-0.3,3.8-0.2,4.2c-1,1.1-1.6,2.5-1.6,4.3c0,6.1,3.7,7.5,7.3,7.9 c-0.5,0.4-0.9,1.1-1,2.1c-0.9,0.4-3.2,1.1-4.7-1.3c0,0-0.8-1.5-2.5-1.6c0,0-1.6,0-0.1,1c0,0,1,0.5,1.8,2.3c0,0,0.9,3.1,5.4,2.1 c0,1.3,0,2.3,0,2.7c0,0.4-0.3,0.9-1.1,0.8C20.6,45.1,16,39.1,16,32c0-8.8,7.2-16,16-16c8.8,0,16,7.2,16,16 C48,39.1,43.4,45.1,37.1,47.2z"
                  />
                </g>
              </svg>
            </div>
          </a>
        </div>
        <div
          class="mt-8 md:order-1 md:mt-0"
        >
          <p
            class="text-center text-base text-gray-400"
          >
            Â© Onu, Inc. All rights reserved.
          </p>
        </div>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
